api/src/main/java/com/thesurvey/api/dto/request/AnsweredQuestionRequestDto.java

```diff
@@ -2,17 +2,18 @@
 
 import java.util.List;
 import java.util.UUID;
-import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotNull;
 import lombok.Builder;
 import lombok.Getter;
 
 @Getter
 @Builder
 public class AnsweredQuestionRequestDto {
 
+    @NotNull(message = "유효하지 않은 설문조사입니다.")
```

Comment:
`message` is good but I think we can catch this error on `GlobalAPIExceptionHandler` which handles error on `non-property` or `null`. 

Well if this property is **required** to be not null or empty string(which is the most strong rule), then you should use `@NotBlank`.

- `@NotBlank`
  - checks empty string (ex. `""`)
  - checks null (ex. `null`)
  - checks blank (ex. `" "`)
- `@NotNull`
  - checks null (ex. `null`)
- `@NotEmpty`
  - checks empty string (ex. `""`)
  - checks blank (ex. `" "`)

So, my opinion overall is to remove `message` since there're `GlobalAPIExceptionHandler` which returns error message, and modify all `required porperties` to `@NotBlank` since it'll catch all 3 conditions.

But which do you think is the right way to do this?

Reply:
I wrote different error messages for each property validation to provide specific error messages,
 but if it's not necessary, I agree that We remove `message`. Or does it mean that we should write it in `GlobalAPIExceptionHandler`??

Upon further research, `ConstraintViolationException` is part of the Java Bean Validation API and is automatically handled by the Spring Boot validation framework, so it seems that the handleValidationException method can be removed.
So, I suggest removing `handleValidationException` method and keeping `message`. 
 
And `@NotBlank` annotation only supports `String` types, so it cannot be applied to `Numeric` or `List` types. 
Instead, I will modify it to use both `@NotNull` and `@NotEmpty` annotations.

Reply:
As you say, I think handling exceptions in `GlobalAPIExceptionHandler` is better. 

While defining messages for each property allows for more specific error messages, it can be cumbersome to write messages for each new property that is added. 
With `GlobalAPIExceptionHandler`, we can respond with error messages that are understandable to the client and allow for flexible exception handling. 

Thank you for your opinion!

Reply:
When I've tested with your code(I'm not 100% for sure), without setting error messages, I could get message from `GlobalAPIExceptionHandler`. So we might need additional messages.

Anyway utilize `Global****` things!